libs/model/src/subscription/RegisterClientRegistrationToken.command.ts

```diff
@@ -0,0 +1,24 @@
+import { notificationsProvider, type Command } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+
+export function RegisterClientRegistrationToken(): Command<
+  typeof schemas.RegisterClientRegistrationToken
+> {
+  const notifications = notificationsProvider();
+  return {
+    ...schemas.RegisterClientRegistrationToken,
+    auth: [],
+    secure: true,
+    body: async ({ payload, actor }) => {
+      if (!actor.user.id) {
```

Comment:
I think this if guard is not needed because we are guaranteed to have the user.id if the route is secure. This is because a secured route requires the jwt to be passed which gets decoded into the user's id. If it gets decoded wrong the passport.deserializeUser will respond to the client with an unauthorized error.

Reply:
Thanks @kurtisassad 

... the id property on User is number | undefined. I'm not sure why that's the case but either way I have to check it here.